stable/snipeit/values.yaml

```diff
@@ -0,0 +1,90 @@
+replicaCount: 1
+revisionHistoryLimit: 0
+
+## PodDisruptionBudget
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget
+# maxUnavailable: 1
+
+image:
+  repository: snipe/snipe-it
+  tag: v4.6.8
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+  annotations: {}
+
+config:
+  mysql:
+  ## All of these values are only used when mysql.enabled is set to false
+    externalDatabase:
+      user: "snipeit"
+      pass: ""
+      name: db-snipeit
+      host: mysql
+      port: 3306
+
+  snipeit:
+    env: production
+    debug: false
+    url: http://example.local
+    key: ""
+    timezone: Europe/Berlin
+    locale: en
+
+mysql:
+  ## Whether to deploy a mysql server to satisfy the applications database requirements. To use an external database set this to false and configure the externalDatabase parameters
+  enabled: false
+  ## Create a database and a database user
+  ##
+  mysqlUser: snipeit
+  mysqlPassword: ""
+  mysqlDatabase: db-snipeit
+
+  persistence:
+    enabled: true
+      ## mysql data Persistent Volume Storage Class
+      ## If defined, storageClassName: <storageClass>
+      ## If set to "-", storageClassName: "", which disables dynamic provisioning
+      ## If undefined (the default) or set to null, no storageClassName spec is
+      ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+      ##   GKE, AWS & OpenStack)
+      ##
+      # storageClass: "-"
+    accessMode: ReadWriteOnce
+    size: 8Gi
+
+persistence:
+  enabled: true
+  annotations: {}
+  accessMode: ReadWriteMany
```

Comment:
ReadWriteMany is something that is usually not supported. Its the app explicitly designed for running with multiple replicas that use the same volume?